scripts/run-mlr-model.py

```diff
@@ -403,15 +429,28 @@ def get_group_samples(samples, sites, group):
         help="Whether to run the model as hierarchical. Overrides model.hierarchical in config. "
         + "Default is false if unspecified."
     )
+
+    parser.add_argument(
+        "--location-ga-inclusion-threshold", type=positive_int, default=0,
+        help="Mininum number of sequences that need to be observed for a specific "
+        + "location x variant combination."
+    )
+
     args = parser.parse_args()
 
     # Load configuration, data, and create model
     config = MLRConfig(args.config)
     print(f"Config loaded: {config.path}")
 
+    # Load sequence data for evofr
     raw_seq, locations = config.load_data(args.seq_path)
     print("Data loaded successfully")
 
+    # Calculate variant x location sequence counts
+    seq_data = pd.read_csv(args.seq_path, sep="\t")
+    variant_location_counts = seq_data.groupby(["location", "variant"])["sequences"].sum().to_dict()
```

Comment:
_non-blocking suggestion_

The `raw_seq` loaded above should be the same Dataframe, so there should be no need to load the data again.
```suggestion
    variant_location_counts = raw_seq.groupby(["location", "variant"])["sequences"].sum().to_dict()
```

Reply:
Thanks for the catch. Fixed in 4ff2370dbdaf5883c963c4de23022cbe11d6d271.